### DISCUSSION TITLE
ibax cüzdan çekimi

### DISCUSSION THREAD

**Discussion by pinarati** [IBAX-io/go-ibax#275] [Rewards]
ibax  nasıl çekeceğiz cüzdandan acaba  zor olacakmı

**Comment by byblckd** [comment]
önce ön satış için bir platform ile anlaşır sonra ön satış yapılır o biter sonra merkeziyetsiz borsalarda yada anlaşılan herhangi bir borsada işlem görmesi için gün belirlenir. ve kendi cüzdanı olacağı için belki de kendi cüzdanın da swap işlemi yaptırır. Bunların hepsi büyük bir muamma zamanla görecez
